packages/api/src/auth/github/github.model.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
What is the problem with our naming convention. I think we should rather update our ESLint config than disable this rule.

Reply:
Eslint at this moment does not accept property name that comes from github: avatar_url

Reply:
In such a case add `// eslint-disable-next-line @typescript-eslint/naming-convention` to disable only the line with wrongly named property.